.github/templates/test-ios.yml

```diff
@@ -7,7 +7,7 @@ name: test-ios
 _: #@ template.replace(testDefinition())
 jobs:
   test-xamarin:
-    runs-on: macos-latest
+    runs-on: macos-12
```

Comment:
Shouldn't we this and the others to 13?

Reply:
13 doesn't have the xamarin tooling. We could manually install it, but I'd rather just use 12 while possible and then either deprecate Xamarin or add a step to manually install at that point.